‘TokenType’ has not been declared

使用最新版本报错，之前版本没有问题 ：
```
/data3/projects//context_cplus_lid_test/onnxruntime/fc_base/wetextprocessing-src/runtime/processor/wetext_processor.cc:17:12: error: ‘TokenType’ has not been declared in ‘fst’
   17 | using fst::TokenType;
      |            ^~~~~~~~~
/data3/projects/zyb/context_cplus_lid_test/onnxruntime/fc_base/wetextprocessing-src/runtime/processor/wetext_processor.cc: In constructor ‘wetext::Processor::Processor(const string&, const string&)’:
/data3/projects/zyb/context_cplus_lid_test/onnxruntime/fc_base/wetextprocessing-src/runtime/processor/wetext_processor.cc:24:56: error: ‘TokenType’ has not been declared
   24 |   compiler_ = std::make_shared<StringCompiler<StdArc>>(TokenType::BYTE);
      |                                                        ^~~~~~~~~
/data3/projects/zyb/context_cplus_lid_test/onnxruntime/fc_base/wetextprocessing-src/runtime/processor/wetext_processor.cc:25:54: error: ‘TokenType’ has not been declared
   25 |   printer_ = std::make_shared<StringPrinter<StdArc>>(TokenType::BYTE);
      |                                                      ^~~~~~~~~
gmake[2]: *** [fc_base/wetextprocessing-src/runtime/processor/CMakeFiles/wetext_processor.dir/build.make:76: fc_base/wetextprocessing-src/runtime/processor/CMakeFiles/wetext_processor.dir/wetext_processor.cc.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:438: fc_base/wetextprocessing-src/runtime/processor/CMakeFiles/wetext_processor.dir/all] Error 2
gmake: *** [Makefile:156: all] Error 2
```

--- a/.github/workflows/runtime.yml
+++ b/.github/workflows/runtime.yml
@@ -1,0 +1,22 @@
+name: Runtime Build
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'runtime/**'
+      - '.github/workflows/runtime.yml'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build runtime
+        run: |
+          cmake -B build -S runtime -DBUILD_TESTING=OFF
+          cmake --build build -j$(nproc 2>/dev/null || sysctl -n hw.ncpu)

--- a/runtime/processor/wetext_processor.cc
+++ b/runtime/processor/wetext_processor.cc
@@ -14,15 +14,15 @@
 
 #include "processor/wetext_processor.h"
 
-using fst::TokenType;
+using fst::StringTokenType;
 
 namespace wetext {
 Processor::Processor(const std::string& tagger_path,
                      const std::string& verbalizer_path) {
   tagger_.reset(StdVectorFst::Read(tagger_path));
   verbalizer_.reset(StdVectorFst::Read(verbalizer_path));
-  compiler_ = std::make_shared<StringCompiler<StdArc>>(TokenType::BYTE);
-  printer_ = std::make_shared<StringPrinter<StdArc>>(TokenType::BYTE);
+  compiler_ = std::make_shared<StringCompiler<StdArc>>(StringTokenType::BYTE);
+  printer_ = std::make_shared<StringPrinter<StdArc>>(StringTokenType::BYTE);
 
   if (tagger_path.find("zh_tn_") != tagger_path.npos) {
     parse_type_ = ParseType::kZH_TN;

--- a/runtime/processor/wetext_processor.cc
+++ b/runtime/processor/wetext_processor.cc
@@ -14,15 +14,13 @@
 
 #include "processor/wetext_processor.h"
 
-using fst::TokenType;
-
 namespace wetext {
 Processor::Processor(const std::string& tagger_path,
                      const std::string& verbalizer_path) {
   tagger_.reset(StdVectorFst::Read(tagger_path));
   verbalizer_.reset(StdVectorFst::Read(verbalizer_path));
-  compiler_ = std::make_shared<StringCompiler<StdArc>>(TokenType::BYTE);
-  printer_ = std::make_shared<StringPrinter<StdArc>>(TokenType::BYTE);
+  compiler_ = std::make_shared<StringCompiler<StdArc>>();
+  printer_ = std::make_shared<StringPrinter<StdArc>>();
 
   if (tagger_path.find("zh_tn_") != tagger_path.npos) {
     parse_type_ = ParseType::kZH_TN;

--- a/runtime/processor/wetext_processor.cc
+++ b/runtime/processor/wetext_processor.cc
@@ -14,15 +14,15 @@
 
 #include "processor/wetext_processor.h"
 
-using fst::StringTokenType;
+using fst::TokenType;
 
 namespace wetext {
 Processor::Processor(const std::string& tagger_path,
                      const std::string& verbalizer_path) {
   tagger_.reset(StdVectorFst::Read(tagger_path));
   verbalizer_.reset(StdVectorFst::Read(verbalizer_path));
-  compiler_ = std::make_shared<StringCompiler<StdArc>>(StringTokenType::BYTE);
-  printer_ = std::make_shared<StringPrinter<StdArc>>(StringTokenType::BYTE);
+  compiler_ = std::make_shared<StringCompiler<StdArc>>(TokenType::BYTE);
+  printer_ = std::make_shared<StringPrinter<StdArc>>(TokenType::BYTE);
 
   if (tagger_path.find("zh_tn_") != tagger_path.npos) {
     parse_type_ = ParseType::kZH_TN;

